recipes/dcv_install.rb

```diff
@@ -15,11 +15,18 @@
 # See the License for the specific language governing permissions and limitations under the License.
 
 # Utility function to install a list of rpm packages
-def install_rpm_packages(packages)
+def install_list_packages(packages)
   packages.each do |package_name|
-    package package_name do
-      action :install
-      source package_name
+    case node['platform']
+    when 'centos'
+      package package_name do
+        action :install
+        source package_name
+      end
+    when 'ubuntu'
+      execute 'apt install dcv package' do
+        command "apt -y install #{package_name}"
```

Comment:
Just for reference why `source` is not working for `apt`
see https://github.com/chef/chef/issues/9014 and https://github.com/chef/chef/issues/5760